### PR TITLE
Update to 0.4.5

### PR DESCRIPTION
**Pull request by futurepaul**
note: I have a version of delete everything (the one that is in the emergency kit) that won't call the new delete function because it's assuming there's not mutiny_wallet loaded, or mutiny_wallet is in a bad state...

(includes #401 and #405 cherry-picked)

### CODE REVIEW

**Comment by TonyGiorgio** [comment]
I just added the env variables in cloudflare for the scorer, so it's not reflected here but everything else seems to work properly.

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
         "@kobalte/tailwindcss": "^0.5.0",
         "@modular-forms/solid": "^0.18.0",
         "@mutinywallet/barcode-scanner": "5.0.0-beta.3",
-        "@mutinywallet/mutiny-wasm": "0.4.4",
+        "@mutinywallet/mutiny-wasm": "0.4.5",
```
Comment (TonyGiorgio) [inline package.json:47]: Needs updating for the package versioning too

Comment (futurepaul) [reply package.json:47]: nice catch

**Review by TonyGiorgio — CHANGES_REQUESTED** [review]
oh actually so all the github actions for the android builds need to be updated too now for each env change: 

https://github.com/MutinyWallet/mutiny-web/blob/7c9c99208464aa923bc765295bfec8240aade8ab/.github/workflows/android-build.yml#L63-L71

File: .github/workflows/android-build.yml [hunk 2]
```diff
@@ -69,6 +69,7 @@ jobs:
           VITE_AUTH: https://auth-staging.mutinywallet.com
           VITE_SUBSCRIPTIONS: https://subscriptions-staging.mutinywallet.com
           VITE_STORAGE: https://storage-staging.mutinywallet.com
+          VITE_SCORER: https://scorer-staging.mutinywallet.com
```
Comment (TonyGiorgio) [inline .github/workflows/android-build.yml:72]: this is missing too: 

```
VITE_FEEDBACK="https://feedback-staging.mutinywallet.com"
```

**Review by TonyGiorgio — CHANGES_REQUESTED** [review]

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**